lib/sla.js

```diff
@@ -116,6 +116,8 @@ function originateChannel(client, inbound, bridge, trunkData) {
   function dialedEnteredStasis(event, line) {
     var answer = Q.denodeify(line.answer.bind(line));
     answer();
+    var deviceState = Q.denodeify(client.deviceStates.update.bind(client));
```

Comment:
Note that this comment applies throughout this review. You've got a lot of code bunched up, without much in the way marking logical separation between actions being taken.